src/main/java/command/sch/SearchCommand.java

```diff
@@ -27,171 +27,233 @@ public List<Employee> run(UserRequest request, EmployeeDB db) {
         List<Employee> resultEmpList = new ArrayList<>();
 
         if (isSearchCmd(request)) {
```

Comment:
if문의 depth를 줄이기 위해서 
```suggestion
        if (!isSearchCmd(request)) {
            return resultEmpList ;
        }
        {
```
이렇게 변경하면 어떨까요? :)